### ISSUE TITLE
How does opensplice perform WAN transmission？

### ISSUE THREAD

**Issue by ZTY-star** [ADLINK-IST/opensplice#133]
Need to configure or install software? How do I configure it? Where do I download the software? Need to spend money?

**Comment by vivekpandey02** [comment]
Vortex Link provides the WAN connectivity for Vortex DDS systems or any other system/device that uses a compliant Data Distribution Service (DDS) software stack. Vortex Link provides transparent discovery and routing between data readers and data writers regardless of location. Vortex Link supports a number of different deployment and connectivity scenarios, including individual device to cloud, system to cloud and also connecting your different LAN’s to turn them into a single system. You can get details from below link:
https://www.adlinktech.com/en/vortex-link-data-distribution-service

**Comment by ZTY-star** [comment]
> 
> 
> **Vortex Link** provides the WAN connectivity for Vortex DDS systems or any other system/device that uses a compliant Data Distribution Service (DDS) software stack. Vortex Link provides transparent discovery and routing between data readers and data writers regardless of location. Vortex Link supports a number of different deployment and connectivity scenarios, including individual device to cloud, system to cloud and also connecting your different LAN’s to turn them into a single system. You can get details from below link:
> https://www.adlinktech.com/en/vortex-link-data-distribution-service

Where can I download the app and do I need to pay for it?

**Comment by vivekpandey02** [comment]
Yes, You need to pay for Vortex Link because it is part of commercial release.

You can download the "Vortex Evaluation Bundle" for link using following link:
https://www.adlinktech.com/en/vortex-data-distribution-service-dds-software-evaluation
